.github/workflows/_snap-build.yaml

```diff
@@ -0,0 +1,18 @@
+on:
+  workflow_call:
+  workflow_dispatch:
+
+jobs:
+  build:
+    name: Build Snap
+    runs-on: ubuntu-latest
+    timeout-minutes: 15
+    steps:
+      - uses: actions/checkout@v3
+      - uses: snapcore/action-build@v1
```

Comment:
note: For future (not in this PR) we may need to implement additional inputs here, like `path` or `snapcraft-channel`.